.gitignore

```diff
@@ -32,6 +32,7 @@
 *.patch
 *.xslt
 *.mbox
+.gitignore
```

Comment:
I don't think it's a good idea to add the .gitignore file in itself, is it? Please remove this commit.